src/interfaces.ts

```diff
@@ -1,5 +1,13 @@
 import { type sources } from 'webpack';
 
+export interface CacheItem {
```

Comment:
webpack has types for it, so you can avoid such things

Reply:
Hum, the name of this interface is probably confusing..

- Before: https://github.com/yoriiis/svg-chunk-webpack-plugin/pull/8/commits/1f30f95550c3fff7c849ae32d3de5d3166526def#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R120-R126
- After: https://github.com/yoriiis/svg-chunk-webpack-plugin/pull/8/commits/098c8c3c9b5a8b480299b104371597ebb7c76bcf#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R132

Reply:
Oh, yes, it is just confusing :smile:

Reply:
Renamed in 4eff8b36083e617c25a395f5d6dd9dd72872dfb5